macros/get_query.sql

```diff
@@ -33,7 +33,14 @@
             {% set account_passthrough_metric_array_of_dicts = var('ad_reporting__account_passthrough_metrics') %}
                 {%- for account_passthrough_metric_dict in account_passthrough_metric_array_of_dicts -%}
                     {%- for account_passthrough_metric_value in account_passthrough_metric_dict.values() -%}
-                        {%- do final_fields_superset.update({account_passthrough_metric_value: account_passthrough_metric_value}) -%}
+
+                        {#- Add _c suffix if conversions are present in the passthrough metrics -#}
+                        {%- if account_passthrough_metric_value == 'conversions' or account_passthrough_metric_value == 'conversions_value' -%}
+                            {%- do final_fields_superset.update({account_passthrough_metric_value ~ '_c': account_passthrough_metric_value}) -%}
+                        {%- else -%}
+                            {%- do final_fields_superset.update({account_passthrough_metric_value: account_passthrough_metric_value}) -%}
+                        {%- endif -%}
```

Comment:
I kinda wonder if macrofying it would make the code less readable in this case. When I was first trying to understand the `get_query` code, I found it helpful to follow `final_fields_superset` and see how it was being updated. Open to other opinions of course

I do however think we could stand to do some higher-level macrofying here, as we [repeat](https://github.com/fivetran/dbt_ad_reporting/blob/main/macros/get_query.sql#L28-L42) a lot of code for each section of `report_types`. Reformatting at that level is probably out of scope for this PR however 

@fivetran-joemarkiewicz

Reply:
Thanks for the explanation here @fivetran-jamie. In this case I think it should be fine to keep the code as is, but it would be good to open a FR for us to explore making this more macro-fied in the future. But we should consider the pros/cons as we don't want to over macro-fy things to the point where it's difficult to maintain. 

We can have that discussion in the FR once you open it.